src/plugins/subsystems/drm/drm.sh

```diff
@@ -345,8 +347,32 @@ function get_supported_mode_per_connector()
   modes=${modes//\/modes/}
   modes=${modes//sys\/class\/drm\//}
 
+  # get current resolution and hz
+  regex='mode: "[0-9]+x[0-9]+": [0-9]+'
+
   say 'Modes per card'
-  printf '%s\n' "$modes"
+  if [[ "$USER_ID" -eq 0 ]]; then
+    for file in "$SYSFS_KERNEL_DEBUG_DRI/"*"/state"; do
+      matches=$(grep -E "${regex}" "$file")
+      while IFS= read -r line; do
+        if [[ $line =~ $regex ]]; then
+          resolution=$(sed -n 's/.*"\(.*\)".*/\1/p' <<< "${BASH_REMATCH[0]}")
```

Comment:
Add a simple comment before your sed that describes (i.e., add an example) what kind of output we expected for `BASH_REMATCH`.